scripts/dist.ts

```diff
@@ -175,7 +177,8 @@ function toDist(sourcePath: string): YAML.Document {
   const compatFeatures = source.compat_features ?? taggedCompatFeatures;
   let computeFrom = compatFeatures;
 
-  if (source.status?.compute_from) {
+  const computeFromWasExplicitlySet = Boolean(source.status?.compute_from);
```

Comment:
I find `hasOwnProperty` less weird than `Boolean()` but 🤷‍♂️

Reply:
Yeah, this is a little weird, but the `Object.hasOwn()` equivalent seems even harder to read, to me at least:

```typescript
const computeFromWasExplicitlySet =
    Object.hasOwn(source, "status") &&
    Object.hasOwn(source.status, "computeFrom");
```

I ended up going with:

```typescript
const computeFromWasExplicitlySet = source.status?.compute_from !== undefined;
```

which seems like the best compromise of readability and conciseness.